tests/unit/test_schemas.py

```diff
@@ -129,12 +132,82 @@ def getDefaultInstance(self, cls):
         """
         Returns a new instance with the required values set.
         """
-        tool = avrotools.Creator(cls)
+        tool = prototools.Creator(cls)
         instance = tool.getDefaultInstance()
         return instance
 
 
 class EqualityTest(SchemaTest):
+    def snakeKeyDict(self, item, field):
```

Comment:
I've added a docString here, but in short it converts (recursively) from camelCasedKeys to snake_keys for a dictionary structure to allow JSON-based examples to be pushed into Protobuf objects.